apps/cli/src/command/index.ts

```diff
@@ -14,9 +14,10 @@ export const setupCommands = (): Command => {
   const program = new Command('adc');
 
   program
+    .description('API Declarative CLI (ADC) is a utility to manage API7 Enterprise and Apache APISIX declaratively.\n\nLearn more at: https://docs.api7.ai/enterprise/reference/adc')
```

Comment:
Does the Enterprise documentation also contain a link on how to use ADC for APISIX? 🤔

Reply:
It does not. Right now, they are mostly similar. Both APISIX and EE have separate ADC docs.